hashintel/hash
# Add visual tweaks to HASH app

## 🌟 What is the purpose of this PR?
This PR adds various visual tweaks to improve the current state of the HASH app. This includes fixing style issues caused by turning off tailwind preflight in an earlier PR, commenting out half implemented buttons and updating some sections to match our designs




## 🔗 Related links





- [Fix appearance and remove half-implemented features in HASH](https://app.asana.com/0/1202176479837585/1202229645898389) _(internal)_

## 🔍 What does this change?




- Merges the icons for reordering and opening BlockContextMenu into 1  to better match designs. see attached Demo
- Fixes style issues, caused by turning of tailwind preflight, especially in auth screens
- Comment out half implemented buttons from Header, Sidebar and Main sections.
- Adds a helper text about names being in Pascal Case to entity type creator. Also ensures the appropriate error is displayed when entity type creation fails
- Display "None" when there no entities in the Entity Type screen instead of an empty string

## 🛡 What tests cover this?



- updated Playwright tests

## ❓ How to test this?



1.  Test signup/login flow, ensure all buttons/inputs look good
1.  Login to the dashboard and confirm all buttons (including those in menus) work. This PR removes those that don't
1. Confirm the new block handle works properly (i.e you can open the block context menu by clicking on it and reorder the block by dragging it)

## 📹 Demo




https://user-images.githubusercontent.com/16928688/169309574-23a36b9a-89df-4fac-9df7-726fe0781c3f.mov


https://user-images.githubusercontent.com/16928688/169308369-33fa3082-d73f-41b2-b6a3-e4a731a906d6.mov


![Screenshot 2022-05-19 at 14 49 19](https://user-images.githubusercontent.com/16928688/169309281-cfbf389c-116f-4039-83da-83f37e0a470b.png)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202229645898389
